### ISSUE TITLE
simplify acquiring result

### ISSUE THREAD

**Issue by DickBaker** [dotnet/docs#34648]
The various code blocks include the line
  total += await finishedTask;
to read the finished int result (or fire any exception), but this is unduly complex and forces another leg to the state machine. Just using
  total += finishedTask;.Result;
is simpler and equally caters for the happy or exception outcomes [i.e. can wrap in Try-catch if needed]
KISS rules !
---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 0ac75088-3781-d9c9-913e-e733cafaa83c
* Version Independent ID: 6f1fbd82-78ca-aa11-481d-4f850e228010
* Content: [Process asynchronous tasks as they complete](https://learn.microsoft.com/en-us/dotnet/csharp/asynchronous-programming/start-multiple-async-tasks-and-process-them-as-they-complete)
* Content Source: [docs/csharp/asynchronous-programming/start-multiple-async-tasks-and-process-them-as-they-complete.md](https://github.com/dotnet/docs/blob/main/docs/csharp/asynchronous-programming/start-multiple-async-tasks-and-process-them-as-they-complete.md)
* Product: **dotnet-csharp**
* Technology: **csharp-async**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

**Comment by BillWagner** [comment]
Hi @DickBaker 

I'm going to close this without action. 

In this example, it would be safe to use the `Result` property, it's discouraged as a general practice. `Result` is synchronous, and blocks the current thread until the `Task` is completed. That can cause deadlocks when misused. 

The recommended practice is to use `await` to asynchronously retrieve the result of a `Task`.

**Comment by DickBaker** [comment]
agreed .Result sometimes bad, but this is EXACTLY the time to use .Result when completion has already been fully realised and the docs code complexifies the state-machine [as already said]. IMHO bad code makes bad docs,
